src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package io.github.externschool.planner.repository.profiles;
+
+import io.github.externschool.planner.entity.profile.Gender;
+import io.github.externschool.planner.entity.profile.Student;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class StudentRepositoryTest {
+
+    @Autowired
+    private StudentRepository studentRepository;
+
+    @Autowired
+    TestEntityManager entityManager;
+
+    @Test
+    public void shouldReturnListOfStudents(){
+        Student student1 = new Student();
+        student1.setFirstName("Marina");
+        student1.setGender(Gender.FEMALE);
+        entityManager.persist(student1);
+
+        Student student2 = new Student();
+        student2.setFirstName("Vasia");
+        student2.setGender(Gender.MALE);
+        entityManager.persist(student2);
+
+        List<Student> studentList = this.studentRepository.findAll();
+
+        System.out.println(studentList.get(0));
```

Comment:
it's redudant

Reply:
as i know printing in a test helps a lot while you're working on it but when it's done nobody reads what you've printed out there, your test passes or fails, that's all.